### PR TITLE
Adds catalog-info.yaml and updates README as per OEP-55

### PR DESCRIPTION
**Pull request by pomegranited**
**Description:** 

Following the steps in [How to maintain a repo](https://docs.openedx.org/en/latest/developers/how-tos/maintain-a-repo.html), this PR:

* adds catalog-info.yaml to record maintainership in Backstage
* updates README.rst to use current cookiecutter format.
* <strike>moves LICENSE to LICENSE.txt to match cookiecutter defaults.</strike>
* Fixes quality issues in CHANGELOG
* fixes bugs to allow use of manage.py shell
* minor updates to PR template

**JIRA:** 

Private-ref: [FAL-3684](https://tasks.opencraft.com/browse/FAL-3684)

**Testing instructions:**

To test this change, please follow the updated [README#getting-started-with-development](https://github.com/openedx/completion/blob/jill/maint-cleanup/README.rst#getting-started-with-development).

**Author Notes & Concerns:**

This repo has [`docs`](https://github.com/openedx/completion/tree/master/docs), but they're not published to docs.openedx.org yet.

Could someone with access please add them? Procedure noted here: https://openedx.atlassian.net/wiki/spaces/DOC/pages/21627535/Publish+Documentation+on+Read+the+Docs

**Merge checklist:**
- [x] All reviewers approved
- [x] CI build is green -- enough. Can't codecov this change without code changes.
- [x] Version bumped - N/A
- [x] Changelog record added - N/A
- [x] Documentation updated (not only docstrings)
- [x] Commits are squashed

**Post merge:**
- [x] Create a tag - N/A
- [x] Check new version is pushed to PyPi after tag-triggered build is finished. - N/A
- [x] Delete working branch (if not needed anymore)

### CODE REVIEW

**Comment by openedx-webhooks** [comment]
Thanks for the pull request, @pomegranited! Please note that it may take us up to several weeks or months to complete a review and merge your PR.

Feel free to add as much of the following information to the ticket as you can:
- supporting documentation
- [Open edX discussion forum threads](https://discuss.openedx.org/)
- timeline information ("this must be merged by XX date", and why that is)
- partner information ("this is a course on edx.org")
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will be done via the GitHub pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).


Please let us know once your PR is ready for our review and all tests are green.

File: README.rst [hunk 1]
```diff
@@ -1,102 +1,133 @@
...
+|pypi-badge| |ci-badge| |codecov-badge| |doc-badge| |pyversions-badge|
+|license-badge| |status-badge|
 
-A library for tracking completion of blocks by learners in edX courses.
+Purpose
+*******
 
-Overview
-________
+A library for tracking completion of blocks by learners in edX courses.
```
Comment (sarina) [inline README.rst:10]:
```suggestion
A library for tracking completion of blocks by learners in Open edX courses.
```

File: README.rst [hunk 2]
```diff
@@ -1,102 +1,133 @@
...
-#. That's it!  In order to simulate a given tox environment ``(django18, django111, quality)``, run ``tox -e <env>`` for the env in question.  If you want to run ``pytest`` directly::
+Documentation
+=============
 
-    pytest completion/tests/test_models.py
+Start by going through `the documentation`_ (generated from `/docs </docs/index.rst>`_).  If you need more help see below.
 
-Getting Help
-------------
+.. _the documentation: https://docs.openedx.org/projects/completion
```
Comment (sarina) [inline README.rst:73]: This link doesn't work for me

File: README.rst [hunk 3]
```diff
@@ -1,102 +1,133 @@
...
-Have a question about this repository, or about Open edX in general?  Please
-refer to this `list of resources`_ if you need any assistance.
+License
+*******
+
+The code in this repository is licensed under version 3 of the AGPL unless
+otherwise noted.
 
-.. _list of resources: https://open.edx.org/getting-help
+Please see `LICENSE.txt <LICENSE.txt>`_ for details.
```
Comment (sarina) [inline README.rst:81]: I saw you renamed [LICENSE → LICENSE.txt](https://github.com/openedx/completion/pull/287/files#diff-d0ed4cc3fb70489fe51c7e0ac180cba2a7472124f9f9e9ae67b01a37fbd580b7) however spot checking a bunch of our other repos I'm only seeing the file called `LICENSE`. A very quick Google seems to say this legally shouldn't matter, but if there wasn't a real good reason to make that switch I'd revert it.

**Review by sarina — APPROVED** [review]
A couple suggestions, merge when you're ready.

File: README.rst (hunk 2 above)
Comment (pomegranited) [reply README.rst:73]: Ya -- I need someone with access please add them? Procedure noted here: https://openedx.atlassian.net/wiki/spaces/DOC/pages/21627535/Publish+Documentation+on+Read+the+Docs

(same issue for edx-submissions, btw)

File: README.rst (hunk 3 above)
Comment (pomegranited) [reply README.rst:81]: Ah ok, will do.

File: README.rst (hunk 2 above)
Comment (sarina) [reply README.rst:73]: Hmm, that page is archived and I don't know the current procedure. @feanil can you help out here?

**Pull request merged**